tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -67,7 +67,6 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
     replicaConfig.clientBatchingEnabled = true;
     replicaConfig.pruningEnabled_ = true;
     replicaConfig.numBlocksToKeep_ = 10;
-    replicaConfig.timeServiceEnabled = true;
```

Comment:
I don't think we should disable time service in concordbft.
Our issues with the new time service were with the product. I think that in order to keep it updated and not break it in concordbft, we should leave it enabled

Reply:
@yontyon  Time service is enabled in concord-bft. Its enabled using the -f parameter in the testcase. After the the timeservice v2 is fully implemented and tested, it will be enabled by default. I am code reviewing that part, so I will make sure that its switched on for all. Now its switched on for skvbc_reconfiguration as it requires time service and its tested.